crates/forge/src/init.rs

```diff
@@ -75,6 +78,39 @@ fn overwrite_files_from_scarb_template(
     Ok(())
 }
 
+fn create_deployment_script(base_path: &Path, project_name: &str) -> Result<()> {
+    let scripts_dir = base_path.join("scripts");
+    let deploy_dir = scripts_dir.join(DEPLOY_PACKAGE_NAME);
```

Comment:
why not move the dir altogether in one move?
`overwrite_files_from_scarb_template` is used for other directories, i think it fits the use case too

Reply:
Not sure if `overwrite_files_from_scarb_template` supports nested dirs, so it might require creating `scripts/deploy` and then `scripts/deploy/src` then calling `overwrite_files_from_scarb_template` for both of them correctly. 

But I think it's still better than duplicating the logic.